src/wavesurfer.js

```diff
@@ -505,7 +505,9 @@ export default class WaveSurfer extends util.Observer {
                     !this.params.scrollParent
                 ) {
                     prevWidth = this.drawer.wrapper.clientWidth;
-                    this.drawer.fireEvent('redraw');
```

Comment:
can you add a comment why you add this condition?

Reply:
I've added a comment